lib/paperclip/storage/s3.rb

```diff
@@ -179,16 +179,16 @@ def sanitize_hash(hash)
         end
 
         Paperclip.interpolates(:s3_alias_url) do |attachment, style|
-          "#{attachment.s3_protocol(style, true)}//#{attachment.s3_host_alias}/#{attachment.path(style).gsub(%r{\A/}, "")}"
+          "#{attachment.s3_protocol(style, true)}//#{attachment.s3_host_alias}/#{attachment.path(style).sub(%r{\A/}, "".freeze)}"
```

Comment:
Line is too long. [129/80]<br>Prefer single-quoted strings inside interpolations.